docs/architecture/ADR-007-bad-debt.md

```diff
@@ -0,0 +1,92 @@
+# ADR 005: Liquidation
+
+## Changelog
+
+- December 10, 2021: Initial Draft (@toteki)
+
+## Status
+
+Proposed
+
+## Context
+
+Debt positions on Umee always start out over-collateralized, but if sufficient interest accrues or asset prices fluctuate too quickly, some borrowers may reach a state where the total value of their collateral is less than the value of their borrowed assets.
+
+Such debt (now under-collateralized) may become over-collateralized on its own if asset prices rebound in the right direction, but it is also eligible for liquidation in its current state.
+When fully liquidated, undercollateralized debt will result in an account with nonzero total borrows but zero collateral, thus no incentive for repayment.
```

Comment:
```suggestion
When fully liquidated, accounts whose `BorrowedValue` is greater than `CollateralValue` (adjusted for liquidation incentive) will result in an account with nonzero total borrows but zero collateral, thus no incentive for repayment.
```
This clarifies that they are not just past their borrow limit, but exceeding their collateral value.